aienvs/Sumo/SumoGymAdapter.py

```diff
@@ -27,30 +25,43 @@ class SumoGymAdapter(Env):
         to connect to SUMO is already in use.
     """
     _DEFAULT_PARAMETERS = {'gui':True,  # gui or not
+                'scenarios_path': os.path.join(os.path.dirname(__file__), "../../scenarios/Sumo/"),
                 'scene':'four_grid',  # subdirectory in the aienvs/scenarios/Sumo directory where
-                'tlphasesfile':'cross.net.xml',  # file
+                'tlphasesfile': None,  # Use None to read phases from net file, otherwise only relative name
                 'box_bottom_corner':(0, 0),  # bottom left corner of the observable frame
                 'box_top_corner':(10, 10),  # top right corner of the observable frame
                 'resolutionInPixelsPerMeterX': 1,  # for the observable frame
                 'resolutionInPixelsPerMeterY': 1,  # for the observable frame
                 'y_t': 6,  # yellow time
+                'generate_conf': True,  # for automatic route/config generation
+
+                'route_generation_method': 'undefined', # One of ['legacy', 'randomTrips.py', 'activitygen']
+
+                # Options for 'route_generation_method' 'activitygen'
+                'activitygen_options': [], # e.g. ["--end", endtime]
+
+                # Options for 'route_generation_method' 'randomTrips.py'
+                'trips_generate_options': ["--validate"], # sumo/tools/randomTrips.py additional options. -n, -o already handled!
```

Comment:
Is it a good idea to add options here? If you want to add extra options, "--validate" would get removed.